canonical/ubuntu.com
# Wd 8751 update the pro subscribe page visuals

## Done

- updated /pro/subscribe page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- checkout /pro/subscribe
- Should match the visuals [here](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?type=design&node-id=558%3A5908&mode=design&t=Ix81wAC44BDZwiMr-1)

## Issue / Card

Fixes #[WD-8751](https://warthogs.atlassian.net/browse/WD-8751)

## Screenshots

![image](https://github.com/canonical/ubuntu.com/assets/30973042/b609c714-cb59-4a57-bb19-5ef1690fb864)



## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-8751]: https://warthogs.atlassian.net/browse/WD-8751?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
